### ISSUE TITLE
All modals - click outside to dismiss

### ISSUE THREAD

**Issue by natelee3** [natelee3/new-interiorize#26]
clicking outside modal is usually considered to be a way of dismissing it - doesn't do that for any of them.  This is a bit hard to do, but would be a nice accessibility thing
